cloudcomposerdiff/.gitignore

```diff
@@ -0,0 +1,4 @@
+__pycache__/
+*.pyc
+venv/
+*.egg-info
```

Comment:
I looked into this, but was unable to do so. 

Could you please tell me how to do this? 

Doesn't git need to know about .gitignore to ignore these files ?

Reply:
Instead of removing the file, I decided to just remove line 3, which is what's specific to your environment. I think it's fair that the other lines would want to be ignored by someone when running/maintaining this tool and are not unique to your specific env